appendices/scope.md

```diff
@@ -0,0 +1,72 @@
+- **Geospatial:** Packages focused on the retrieval, manipulation, and analysis of spatial data.
+- **Education:** Packages to aid with instruction.
+- **Data visualization:** Packages for visualizing and analyzing data.
+
+## Package technical scope
+
+To be in technical scope for a pyOpenSci review, your package:
+
+* Should have maintenance workflows documented 
+* Should be structured in a way that someone else could contribute to it
+* Should vendor dependencies using standard approaches rather than including code from other packages within your repository. 
+
+### Notes on scope categories
+- pyOpenSci is still developing as a community. If your scientific Python 
+package does not fit into one of the categories or if you have any other 
+questions, we'd encourage you to open a pre-submission inquiry. We're happy to help.
+- Data visualization packages come in many varieties, ranging from small 
+hyper-specific methods for one type of data to general, do-it-all packages 
+(e.g. matplotlib). pyOpenSci accepts packages that are somewhere in between the 
+two. If you're interested in submitting your data visualization package, please 
+open a pre-submission inquiry on first.
+
+## Python package technical scope
+
+pyOpenSci is may continue to update it's technical scope criteria for package 
+review as more packages with varying structural approaches are reviewed.  
+Your package **may not be in technical scope** for us to review at this time if
+fits any of the out-of-technical-scope criteria listed below. 
+
+```{important}
+**If the code base of your package is exceedingly complex in terms of 
+structure of maintenance needs**, we may not be able to review it. 
+
+pyOpenSci has a goal of supporting long term maintenance of open source 
+Python tools. It is thus important for us to know that if you need to step down as a maintainer, the package infrastructure and documentation is 
+in place to support us finding a new maintainer who can take over you 
+package's maintenance. 
+
+## Examples of technically complex package structures that may be difficult for us to review
+
+### Example 1: Your package is an out of sync fork of another package repository that is being actively maintained. 
+
+Sometimes we understand that a package maintainer may need to step down. In 
+that case, we strongly suggest that the original package owner, transfer the 
+package repository to a new organization along with PyPI credentials. A new 
+organization would allow transfer of ownership of package maintenance rather 
+than several forks existing.
+
+Of your package is a divergent fork of a maintained repository we will encourage you 
+to work with the original maintainers to merge efforts. 
+
+However, if there is a case where a forked repository is warranted, please 
+consider submitting a pre-submission inquiry first and explain why the package is a 
+fork rather than an independent parent repository.
+
+### Example 2: Vendored dependencies
```

Comment:
this is a really good question. So this topic came up with a previous review - https://github.com/pyOpenSci/software-review/issues/60 we had to close the issue and reject because they had forked the repo that they were vendoring and then were writing code on top of that fork. The issue there was multi-faceted but frm our perspective it would be VERY hard to find a new maintainer for that kind of custom workflow. I think if it had been a git submodule it could have been ok.

Its funny this came up as it's really an edge case but i'm learning that i think python will be full of edge cases that R doesn't see as often! 

 technically I don't know how a submodule works for packaging. Can you tell me @Batalex when you implement a submodule and package to tool are you then shipping the tool with both the vendored submodule AND the package itself vs installing as a dependency? many thanks!

Reply:
I have only tried once, so I am no expert.
A git submodule allows you to add secondary git folders inside your project. The commit histories would remain separate, however.

Since API wrappers and bindings for other tools are in scope for pyOpenSci, we could end up in a situation where a python package would ship some native lib in a submodule. Any time you want to release a new version, you can git pull the submodule to get the latest commit.

This usage would be pretty marginal though, especially considering that one may use conda to install non-python dependencies.

Reply:
thank you so much, @Batalex !! your thoughts here are super helpful!